var/spack/repos/builtin/packages/libunwind/package.py

```diff
@@ -24,14 +25,19 @@ class Libunwind(AutotoolsPackage):
     variant('xz', default=False,
             description='Support xz (lzma) compressed symbol tables.')
 
+    variant('zlib', default=False,
```

Comment:
Can you have `+xz+zlib`?

Reply:
> Can you have +xz+zlib?

Yes, that's allowed, at least for develop/master.  For any other
version, +zlib is ignored.